spec/datadog/tracing/contrib/http/circuit_breaker_spec.rb

```diff
@@ -74,45 +74,19 @@
           it { is_expected.to be false }
         end
       end
-    end
-
-    context 'integration' do
```

Comment:
This integration test was moved to `spec/datadog/tracing/contrib/http/request_spec.rb` and refactored.